ethers-solc/tests/project.rs

```diff
@@ -47,22 +39,10 @@ fn can_compile_hardhat_sample() {
 
 #[test]
 fn can_compile_dapp_sample() {
-    let tmp_dir = TempDir::new("root").unwrap();
-    let cache = tmp_dir.path().join("cache");
-    let cache = cache.join(SOLIDITY_FILES_CACHE_FILENAME);
-    let artifacts = tmp_dir.path().join("out");
-
     let root = PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("test-data/dapp-sample");
-    let paths = ProjectPathsConfig::builder()
-        .cache(cache)
-        .sources(root.join("src"))
-        .artifacts(artifacts)
-        .lib(root.join("lib"))
-        .root(root)
-        .build()
-        .unwrap();
+    let paths = ProjectPathsConfig::builder().sources(root.join("src")).lib(root.join("lib"));
+    let project = TempProject::<MinimalCombinedArtifacts>::new(paths).unwrap();
```

Comment:
Maybe this could be done with `TempProject::<_>::dapptools()`?

Reply:
hmm we're essentially setting the project root to `./test-data/..` instead of creating a totally new, empty temp project that `TempProject::<_>::dapptools()` would do.
in these cases we'd need to use `TempProject::new` and pass in the preconfigured ProjectPaths,